### PR TITLE
(DOCS-8222)(Agent) Agent 6 installation

### PR DESCRIPTION
**Pull request by hestonhoffman**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/agent/guide/install-agent-6.md [hunk 1]
```diff
@@ -0,0 +1,1240 @@
...
+   sudo apt-get install apt-transport-https curl gnupg
+   ```
+1. Set up the Datadog Debian repo on your system and create a Datadog archive keyring:
+   ```shell
+   sudo sh -c "echo 'deb [signed-by=/usr/share/keyrings/datadog-archive-keyring.gpg] https://apt.datadoghq.com/ stable 6' > /etc/apt/sources.list.d/datadog.list"
+   sudo touch /usr/share/keyrings/datadog-archive-keyring.gpg
+   sudo chmod a+r /usr/share/keyrings/datadog-archive-keyring.gpg
+
+   curl https://keys.datadoghq.com/DATADOG_APT_KEY_CURRENT.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+   curl https://keys.datadoghq.com/DATADOG_APT_KEY_06462314.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
```
Comment (Pythyu) [inline content/en/agent/guide/install-agent-6.md:192]:
```suggestion
```
This line is duplicated

**Review by Pythyu — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: content/en/agent/guide/install-agent-6.md [hunk 2]
```diff
@@ -0,0 +1,1240 @@
...
+
+{{< /tabs >}}
+
+## Cloud and containers
+
+{{< tabs >}}
+
+{{% tab "Kubernetes" %}}
+
+Run the Datadog Agent in your Kubernetes cluster directly to start collecting your cluster and applications metrics, traces, and logs. You can deploy the Agent with a Helm chart, [the Datadog Operator][101] or directly with [a DaemonSet][102]. For more information about installing the Datadog Agent on different distributions, see the [Kubernetes distributions documentation][103].
```
Comment (estherk15) [inline content/en/agent/guide/install-agent-6.md:691]:
```suggestion
Run the Datadog Agent directly in your Kubernetes cluster to start collecting your cluster and applications metrics, traces, and logs. You can deploy the Agent with a Helm chart, [the Datadog Operator][101] or directly with [a DaemonSet][102]. For more information about installing the Datadog Agent on different distributions, see the [Kubernetes distributions documentation][103].
```

File: content/en/agent/guide/install-agent-6.md [hunk 3]
```diff
@@ -0,0 +1,1240 @@
...
+   ```shell
+   helm install RELEASE_NAME -f datadog-values.yaml --set datadog.site='datad0g.com' --set agents.image.tag='6' --set datadog.apiKey=MY_API_KEY datadog/datadog
+   ```
+
+   **With Helm v1 or v2**:
+   ```shell
+   helm install -f datadog-values.yaml --name RELEASE_NAME --set datadog.site='datad0g.com' --set agents.image.tag='6' --set datadog.apiKey=MY_API_KEY datadog/datadog
+   ```
+
+   This chart adds the Datadog Agent to all nodes in your cluster via a DaemonSet. Soon after installation, Datadog begins to report hosts and metrics data in your account.
```
Comment (estherk15) [inline content/en/agent/guide/install-agent-6.md:720]:
```suggestion
   This chart adds the Datadog Agent to all nodes in your cluster through a DaemonSet. Soon after installation, Datadog begins to report hosts and metrics data in your account.
```

File: content/en/agent/guide/install-agent-6.md [hunk 4]
```diff
@@ -0,0 +1,1240 @@
...
+{{% /tab %}}
+
+{{< /tabs >}}
+
+## Configuration management
+
+{{< tabs >}}
+{{% tab "Ansible" %}}
+
+Installing the Agent via Ansible requires Ansible version 2.10 or higher.
```
Comment (estherk15) [inline content/en/agent/guide/install-agent-6.md:898]:
```suggestion
Installing the Agent through Ansible requires Ansible version 2.10 or higher.
```

File: content/en/agent/guide/install-agent-6.md [hunk 5]
```diff
@@ -0,0 +1,1240 @@
...
+    include "datadog_agent::integrations::elasticsearch"
+}
+```
+
+Refer to the [GitHub repository of the module][103] for more examples and advanced use cases.
+
+[101]: https://forge.puppetlabs.com/modules/datadog/datadog_agent/readme
+[102]: https://github.com/DataDog/puppet-datadog-agent/tree/main/manifests/integrations
+[103]: https://github.com/DataDog/puppet-datadog-agent
+[104]:
```
Comment (estherk15) [inline content/en/agent/guide/install-agent-6.md:1066]: Empty link
```suggestion
```

File: content/en/agent/guide/install-agent-6.md [hunk 6]
```diff
@@ -0,0 +1,1240 @@
...
+[102]: https://github.com/DataDog/chef-datadog
+
+{{% /tab %}}
+
+{{% tab "SaltStack" %}}
+
+<div class="alert alert-info">The Datadog Saltstack formula only supports Debian-based and RedHat-based systems.<br><br>
+The following instructions add the Datadog formula to the <code>base</code> Salt environment. To add it to another Salt environment, replace references to <code>base</code> with the name of your Salt environment.</div>
+
+<!-- vale Datadog.inclusive = NO -->
```
Comment (estherk15) [inline content/en/agent/guide/install-agent-6.md:1131]: Is this supposed to be left here?

**Review by estherk15 — APPROVED** [review]
Mostly small suggestions! Looks good!

File: content/en/agent/guide/install-agent-6.md (hunk 6 above)
Comment (hestonhoffman) [reply content/en/agent/guide/install-agent-6.md:1131]: Yes, it prevents Vale from flagging "master"

**Review by Pythyu — APPROVED** [review]

File: content/en/agent/guide/install-agent-6.md [hunk 7]
```diff
@@ -0,0 +1,1240 @@
...
+   gpgcheck=0
+   repo_gpgcheck=1
+   gpgkey=https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public
+          https://keys.datadoghq.com/DATADOG_RPM_KEY_4F09D16B.public
+          https://keys.datadoghq.com/DATADOG_RPM_KEY_B01082D3.public
+          https://keys.datadoghq.com/DATADOG_RPM_KEY_FD4BF915.public
+          https://keys.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public
+   ```
+
+   **Note**: The `repo_gpgcheck=0` option is a workaround for [a bug in DNF][102].
```
[the remark this answers is not in the document]
Comment (hestonhoffman) [reply content/en/agent/guide/install-agent-6.md:359]: Good point, I'll look into this - don't want to block this PR though

File: content/en/agent/guide/install-agent-6.md [hunk 8]
```diff
@@ -0,0 +1,1238 @@
+---
+title: Install Datadog Agent 6
+kind: Guide
```
Comment (hestonhoffman) [inline content/en/agent/guide/install-agent-6.md:3]:
```suggestion
```

**Pull request merged**